components/OpportunityCard.jsx

```diff
@@ -0,0 +1,146 @@
+import styled from "styled-components";
+import { Text } from "./Text";
+
+export const OpportunityCard = ({
+  name,
+  location,
+  category,
+  organization,
+  postingDate,
+  logo,
+	link,
+  ...props
+}) => {
+  return (
+    <Wrapper {...props}>
+      <Logo src={logo} />
+      <InfoContainer>
+        <Name href={link}>{name}</Name>
+        <Organization>{organization}</Organization>
+        <Category>{category}</Category>
+        <BottomText>
+          <Location>{location}</Location>
+          <PostingDate>Posted {postingDate}</PostingDate>
+        </BottomText>
+      </InfoContainer>
+    </Wrapper>
+  );
+};
+
+const Wrapper = styled.div`
+  ${({ theme }) => `
+		border-color: ${theme.font.text};
+		box-shadow: ${theme.boxShadow.topBottom};
+	`};
+  border-radius: 20px;
+  display: flex;
+  flex-direction: row;
+  height: 130px;
+  padding: 10px;
+  margin: 10px;
+  align-items: center;
+  @media only screen and (max-width: 900px) {
+    height: 100px;
+    padding: 5px;
+  }
+`;
+
+const BottomText = styled.div`
+  margin: 0;
+  padding: 0;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+`;
+
+const Name = styled.a`
+  ${({ theme }) => `
+		font-family: ${theme.font.lato};
+		color: ${theme.colors.navy};
+		font-size: ${theme.size.defaultLarger};
+	`};
+  font-weight: bold;
+  margin: 0;
+  @media only screen and (max-width: 900px) {
+    ${({ theme }) => `
+			font-size: ${theme.size.default};
+		`};
+  }
+`;
+
+const Organization = styled(Text)`
+  ${({ theme }) => `
+		font-family: ${theme.font.lato};
+		font-size: ${theme.size.defaultLarger};
+	`};
+  color: #868e9f;
+  margin: 0;
+  @media only screen and (max-width: 900px) {
+    ${({ theme }) => `
+			font-size: ${theme.size.small};
+		`};
+  }
+`;
+
+const Category = styled(Text)`
+  ${({ theme }) => `
+		font-family: ${theme.font.lato};
+		color: ${theme.colors.navy};
+	`};
+  font-size: 1.2rem;
+  text-align: right;
+  margin: 0;
+  @media only screen and (max-width: 900px) {
+    ${({ theme }) => `
+			font-size: ${theme.size.small};
+		`};
+    margin-top: 5px;
+  }
+`;
+
+const Location = styled(Text)`
+  ${({ theme }) => `
+		font-family: ${theme.font.lato};
+	`};
+  font-size: 1.2rem;
+  color: #868e9f;
+  margin: 0;
+  @media only screen and (max-width: 900px) {
+    ${({ theme }) => `
+			font-size: ${theme.size.small};
+		`};
+  }
+`;
+
+const PostingDate = styled(Text)`
+  ${({ theme }) => `
+		font-family: ${theme.font.lato};
+	`};
+  font-size: 1.2rem;
+  color: #868e9f;
+  text-align: right;
+  margin: 0;
+  @media only screen and (max-width: 900px) {
+    ${({ theme }) => `
+			font-size: ${theme.size.small};
+		`};
+  }
+`;
+
+const Logo = styled.img`
```

Comment:
What happens when you use next's Image? The logos are cutting off and I'd rather have the full logo with some white space around it than how it's being cut off now.

Reply:
logos no longer cut off, now they will scale down to fit